### PR TITLE
Clockcult rework chunk A

### PR DESCRIPTION
**Pull request by ChangelingRain**
### welcome to die.

:cl: Joan
balance: Sentinel's Compromise now uses Vitality to heal, and healing done without enough Vitality will be much less effective, converting 75% of the damage healed to toxin damage.
tweak: Vitality is now tracked in the global alert.
balance: Vanguard now lasts for 25 seconds, from 20, stuns for 30% of the stuns absorbed, from 25%, and no longer stuns for double the time if a large amount of stuns were absorbed.
balance: Sigils of Transgression now stun for 3 seconds(if the target has no Belligerent) and apply Belligerent for 7 seconds.
experiment: Ocular Wardens now attack all nearby non-Servants, applying Belligerent and producing small amounts of Vitality.
balance: Changed Ocular Warden damage per second from 13.5 to 4. Changed Ocular Warden health from 25 to 60. Ocular Wardens will now produce Vitality equal to damage done.
wip: Ocular Wardens will still attack mechs at a rate of 25 damage a second, but will, obviously, not produce Vitality from those attacks.
imageadd: Ocular Wardens have new sprites.
balance: All cyborg Servants now have a cyborg fabricator, a ratvarian spear, and a normal, not-limited slab that has two cyborg-unique scriptures bound.
balance: Cyborg Servants casting Geis will have bright purple eyes.
rscadd: Replaces Function Call and Fellowship Armory with Arcane Armory, a script scripture with the same cost as Function Call.
tweak: Arcane Armory permanently gives you the ability to summon and dismiss a Ratvarian spear(Summon Spear) and a set of clockwork armor(Call Cuirass). These abilities will follow you across mind transfers such as cloning, though you will only receive them if you are able to hold the spear.
balance: Ratvarian spears now last until dismissed or until thrown at someone and broken. Breaking a Ratvarian spear by throwing it at someone puts the Summon Spear action on a 30 second cooldown.
balance: Being hit with clockwork armor on will increase the duration of Vanguard slightly; 1.5 seconds with a full set of armor.
balance: The Ark of the Clockwork Justicar no longer needs to consume 15 of each component after being summoned to properly activate. Changed invokers required to summon the Ark from 6 to 3.
tweak: Clockwork Obelisks are now a Script rather than an Application, and their cost has been accordingly adjusted.
balance: Changed Obelisk gateway cost from 2000W to 1500W.
tweak: Spatial Gateway is now a Driver rather than an Application, and the cost has been accordingly adjusted.
balance: Spatial Gateway can now target and teleport to the Ark of the Clockwork Justicar.
tweak: Judicial Visors are now a Script rather than a Driver, and their cost has been accordingly adjusted.
tweak: Wraith Spectacles have had their vision decay rate massively reduced and their cost has been accordingly adjusted.
tweak: Tweaked component requirements for several scriptures.
/:cl:

Cyborgs are rapidly losing a lot of tools so let's just make them actual Servants instead of super weak.
Effectively, every cultist has personal armor and a weapon with a bunch of synergy.
Ark change: A Robustin™️ sponsored idea.
More chances to try to summon, more chances to fuck it up, less completely donging over the cult if you mess it up.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
I put up a player poll about this vs new cult

**Comment by ChangelingRain** [comment]
These aren't actually mutually exclusive though.
I want to get and the followup, B, in, since these are a better base for xhuis(or me) to do reebecult with.

**Comment by Super3222** [comment]
it's like the time when two coders fought over bloodcult

**Comment by Xhuis** [comment]
Better base or not, additional changes conflict with the rework currently up, and should they get merged it will cause conflicts and problems in getting the new changes ironed out. The best thing for me to do with regards to the rework is to cherry-pick the changes that would work well for it.

**Comment by ChangelingRain** [comment]
The hardest part of the rework is mapping reebe, and I can do the rest on this base.

**Comment by Xhuis** [comment]
The hardest part of the rework is making the concept work in a fun way for both sides. While the cooperation is either way a positive thing for the mode as a whole (regardless of whether or not the rework gets in), making a gamemode not suck for one side is far harder than you seem to perceive it as.

**Comment by PopNotes** [comment]
I'm not a fan of the Ocular Warden nerf. The damage nerf is way, way too much compared to the itsy bitsy health buff.

**Comment by DoctoreMad** [comment]
~TO DO list: Nerf Clockwork cult~

**Comment by ChangelingRain** [comment]
Ocular wardens are meant to have the cult follow up on them as defenders rather than having the ocular wardens themselves be the defense.

**Comment by Armhulen** [comment]
TO DO list: Remove Clockwork cult if the new one doesn't get in

**Comment by Shadowlight213** [comment]
Tldr, give abridged version

**Comment by optimumtact** [comment]
Of all the clockcult reworks, I support the one that substantially changes how it plays so its not just a rethemed cult

I think the xhuis tower defence rework works better for that case so this is just pointless :-1:

**Comment by Ratvar** [comment]
Take a cold shower, Joan

**Comment by AdamElTablawy** [comment]
Ocular wardens seem significantly nerfed here.

Yes, I see that they are supposed to be supporting defenders. Yes, I see that they are AoE and also apply belligerent.

But aside from the slowdown, the damage they do is incredibly minor. It's likely not to make a difference at all.

If the damage done was 5 or so, then yes, that would certainly be a fair trade off-exchanging something that serves as a fantastic defender on its own but focuses solo targets for a low-damage AoE delay turret, but 2?

2 is piddle damage. I think the fucking SORD does more.

**Comment by ChangelingRain** [comment]
Note: I doubled it to 4 per second in previous commits, so it takes about 25 seconds to kill someone assuming there's nobody at all following up on it.

**Comment by optimumtact** [comment]
same as the other comment

**Comment by optimumtact** [comment]
stale ping for reopen etc etc, or make new pr bla bla you know what you did

**Comment by ChangelingRain** [comment]
@optimumtact neither of these are actually stale, they're both in a ready state
